pkgs/development/interpreters/ruby/default.nix

```diff
@@ -210,6 +210,8 @@ let
           sed -i "s|'--with-baseruby=${baseruby}/bin/ruby'||" $rbConfig
         '';
 
+        disallowedReferences = op (!jitSupport) stdenv.cc;
```

Comment:
Maybe:

```suggestion
        disallowedReferences = lib.optionals (!jitSupport) [ stdenv.cc stdenv.cc.cc ];
```

Reply:
Then I propose

```
        disallowedRequisites = op (!jitSupport) stdenv.cc.cc;
```

Since that checks the transitive closure and `stdenv.cc` will have `stdenv.cc.cc` in its closure.